python/cudf/cudf/tests/test_index.py

```diff
@@ -1866,7 +1873,12 @@ def test_index_fillna(data, fill_value):
     pdi = pd.Index(data)
     gdi = cudf.Index(data)
 
-    assert_eq(pdi.fillna(fill_value), gdi.fillna(fill_value))
```

Comment:
We could just use `check_dtypes=False` here too.

Reply:
`assert_index_equal` doesn't seem to be having that parameter. The closest thing that skips `type` checking is `exact` param: https://pandas.pydata.org/docs/reference/api/pandas.testing.assert_index_equal.html